configuration/reports/reportdescriptors/dataexports/sql/pathology.sql

```diff
@@ -1,141 +1,208 @@
+-- set @startDate = '2022-03-28';
```

Comment:
The rest of this is impossible to review for diffs.  Not looking in detail.  Let me know if anything in particular needs a look prior to testing.

Reply:
no, it should be fine.  All of this is now being generated from standard functions.